src/app/collective-rewards/components/BackersCallToAction/BackersCallToAction.tsx

```diff
@@ -0,0 +1,72 @@
+import { Header, Paragraph } from '@/components/TypographyNew'
+import { MetricsContainer } from '@/components/containers'
+import { Button } from '@/components/ButtonNew/Button'
+import { useContext } from 'react'
+import { useAccount } from 'wagmi'
+import { Banner } from '../Banner'
+import { BackersDecorativeSquares } from '../DecorativeSquares'
+import { AllocationsContext } from '../../allocations/context'
+import { CallToActionCard } from '../CallToActionCard'
+import { ActiveBackers } from '../ActiveBackers/ActiveBackers'
+import { FC } from 'react'
+import { RewardsMetrics } from '../RewardsMetrics'
+
+const BackersBanner = () => (
+  <Banner
+    imageSrc="/images/cta-banner-backers.png"
+    altText="Backers Call to Action Banner"
+    DecorativeComponent={BackersDecorativeSquares}
+  />
+)
+
+const BackersTitle = () => {
+  return (
+    <Header caps variant="e2" className="px-6 py-4">
+      <div className="text-v3-text-0">Back Builders</div>
+      <div className="text-v3-bg-accent-20">and be rewarded</div>
+    </Header>
+  )
+}
+
+const BackerCTAButton = () => {
+  const { isConnected } = useAccount()
+  const {
+    state: {
+      backer: { balance: votingPower },
+    },
+  } = useContext(AllocationsContext)
+
+  const hasStRIF = votingPower && votingPower > 0n
+
+  if (!isConnected || hasStRIF) {
+    return <Button variant="secondary">Back Builders</Button>
+  }
+
+  return <Button variant="secondary">Stake RIF</Button>
```

Comment:
I don't see this in the design. Was this agreed with UX or product?

Reply:
Is in the user story [TOK-760](https://rsklabs.atlassian.net/browse/TOK-760)